network/src/network.rs

```diff
@@ -713,7 +726,11 @@ impl ServiceHandle for EventHandler {
                         .disconnecting_sessions
                         .write()
                         .insert(session_id);
-                    if let Err(err) = context.disconnect(session_id) {
+                    if let Err(err) = disconnect(
+                        context.control(),
+                        session_id,
+                        Some("already removed from registry"),
```

Comment:
```suggestion
                        Some("peer already disconnect")
```

Reply:
For debug purpose `removed from registry` is better.